pkg/watch/listwatch/until.go

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
Agree. We actually already had a ListWatcher interface definition in client/cache.